optuna/storages/_rdb/alembic/versions/v3.0.0.a.py

```diff
@@ -0,0 +1,224 @@
+"""unify existing distributions to {int,float} distribution
+
+Revision ID: v3.0.0.a
+Revises: v2.6.0.a
+Create Date: 2021-11-21 23:48:42.424430
+
+"""
+from typing import Any
+
+from alembic import op
+from sqlalchemy import Column
+from sqlalchemy import DateTime
+from sqlalchemy import Enum
+from sqlalchemy import Float
+from sqlalchemy import ForeignKey
+from sqlalchemy import Integer
+from sqlalchemy import orm
+from sqlalchemy import String
+from sqlalchemy import Text
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.declarative import declarative_base
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import DiscreteUniformDistribution
+from optuna.distributions import distribution_to_json
+from optuna.distributions import FloatDistribution
+from optuna.distributions import IntDistribution
+from optuna.distributions import IntLogUniformDistribution
+from optuna.distributions import IntUniformDistribution
+from optuna.distributions import json_to_distribution
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna.trial import TrialState
+
+
+MAX_INDEXED_STRING_LENGTH = 512
+
+
+BaseModel = declarative_base()
+
+
+class StudyModel(BaseModel):
+    __tablename__ = "studies"
+    study_id = Column(Integer, primary_key=True)
+    study_name = Column(String(MAX_INDEXED_STRING_LENGTH), index=True, unique=True, nullable=False)
+
+
+class TrialModel(BaseModel):
+    __tablename__ = "trials"
+    trial_id = Column(Integer, primary_key=True)
+    number = Column(Integer)
+    study_id = Column(Integer, ForeignKey("studies.study_id"))
+    state = Column(Enum(TrialState), nullable=False)
+    datetime_start = Column(DateTime)
+    datetime_complete = Column(DateTime)
+
+    study = orm.relationship(
+        StudyModel, backref=orm.backref("trials", cascade="all, delete-orphan")
+    )
```

Comment:
This attribute may be able to omitted since we use `bulk_save_objects` that does not check the relationship. Please refer to `v2.4.0.py`.

```suggestion
```

Reply:
Thank you for the careful look. I removed relations in 3d45cbf.